File: server/job/src/main/java/io/iamcyw/ams/job/strategy/persistence/StrategySolvePO.java
package io.iamcyw.ams.job.strategy.persistence;

import io.quarkus.hibernate.orm.panache.PanacheEntity;

import javax.persistence.Entity;

/**
 * alarm ack 策略
 */
@Entity(name = "ALARM_STRATEGY_SOLVE")
public class StrategySolvePO extends PanacheEntity {

}

File: domain/src/jvmMain/kotlin/io/iamcyw/ams/domain/job/strategy/StrategySolvePO.kt
package io.iamcyw.ams.domain.job.strategy

import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntityBase
import javax.persistence.Entity

/**
 * alarm ack 策略
 */
@Entity(name = "ALARM_STRATEGY_SOLVE")
actual class StrategySolvePO : PanacheEntityBase {

    companion object : PanacheCompanion<StrategySolvePO>
}